src/themes/teams/components/Icon/svg/icons/gallery.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg
+      viewBox="0 0 32 32"
+      role="img"
+      className="app-svg icons-gallery"
```

Comment:
this `className` line should be removed, because it will introduce coupling of this icon with static CSS selector/styles - in case if it is needed, could you, please, suggest the reason for that

Reply:
It is not needed . Removed